package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "smart-round",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
Why changing the major version if the API remains exactly the same? The only change is implementation, not even a new feature. So even a bump to 2.0.1 would work too.

Reply:
The end of the PR description says

> Note: I believe this should be a breaking change because while the output should stay the same, the input used to accept a BigNumber.js instance, which is no longer accepted. We use BigSource now.